wagtailio/services/templates/services/services_page.html

```diff
@@ -3,7 +3,6 @@
 {% load static wagtailcore_tags wagtailimages_tags %}
 
 {% block content %}
-    {% include "services/includes/svg_sprites.html" %}
```

Comment:
Comment: Should we remove the sprites if we're not using them anymore?

Reply:
I noticed that the file `services/includes/svg_sprites.html` seems to have been removed in a376628, and replaced with `patterns/includes/svg_sprites.html`, which is already `include`d in the `base.html` template (#181).

Reply:
Good point, thanks @engineervix.